src/routes/(console)/organization-[organization]/billing/paymentHistory.svelte

```diff
@@ -73,7 +73,7 @@
             <TableScroll noMargin transparent noStyles>
                 <TableHeader>
                     <TableCellHead width={100}>Due Date</TableCellHead>
-                    <TableCellHead width={80}>Status</TableCellHead>
+                    <TableCellHead width={100}>Status</TableCellHead>
```

Comment:
@ernstmul `100` looks okay or do we need for better spacing like `105<>120`ish? 👀

Reply:
Done, moved to `110`.